src/sys/sys_win32_con.c

```diff
@@ -700,12 +700,14 @@ void Sys_CreateConsole(void)
 	RECT     rect;
 
 	const char *DEDCLASS = "ET: Legacy WinConsole";
+
+	char windowname[MAX_STRING_CHARS] = { 0 };
 #if defined (UPDATE_SERVER)
-	const char *WINDOWNAME = "ET: Legacy Update Server";
+	Q_strncpyz(windowname, "ET: Legacy Update Server", sizeof(windowname));
 #elif defined(DEDICATED)
-	const char *WINDOWNAME = "ET: Legacy Dedicated Server";
+	Q_strncpyz(windowname, "ET: Legacy Dedicated Server", sizeof(windowname));
 #else
-	const char *WINDOWNAME = "ET: Legacy Console";
+	Com_sprintf(windowname, sizeof(windowname), "%s Console", GlobalGameTitle);
```

Comment:
This gets called before command line args are parsed, so it doesn't work, but that's really not a critical thing to fix, can be looked at later as it could be rather complex to do.